src/haconiwa/scan/guide_generator.py

```
@@ -0,0 +1,461 @@
+        
+        # Process files
+        for category, files in search_results['matches'].items():
+            for file_info in files:
+                file_path = Path(file_info['path'])
+                
+                # Extract config
+                if file_path.name in ['config.json', 'model_config.json']:
+                    try:
+                        model_info['config'] = json.loads(file_info.get('content', '{}'))
+                    except:
+                        pass
```

Ruff (0.11.9)

Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(Exception)`

(SIM105)